### ISSUE TITLE
Help needed. Train model does not produce good outputs

### ISSUE THREAD

**Issue by sbetzin** [jcjohnson/fast-neural-style#165]
Hi there, I really Need some help: What I am doing wrong? Trained dozen of models but they all give me nearly no effect. starting with the Environment:
Using a Azure GPU Machine with nvidia-docker.

My Image is build with this docker file:

[https://github.com/sbetzin/neural-style-azure/blob/master/docker/neural-style-torch-docker/Dockerfile](https://github.com/sbetzin/neural-style-azure/blob/master/docker/neural-style-torch-docker/Dockerfile)

1. I used the coco 2014 Images and created a coco2014.h5
2. Using vgg16.t7 as loss Network
3. Using this th command line:
`th /app/fast-neural-style/train.lua -h5_file /training/coco2014.h5 -style_image /training/style/gogh_stary_night.jpg -content_weights 1 -style_weights 5  -loss_network /app/fast-neural-style/vgg16.t7 -checkpoint_name /training/models/style/gogh_stary_night -checkpoint_every 500`

using this model on an Image gives me nearly no style transfer:
![result](https://user-images.githubusercontent.com/1607849/41798097-a9033efc-766c-11e8-9243-b30a58619ed5.jpg)

btw: if I using pretrained models everythings works fine! I just cant train any new usefull models

**Comment by gg22mm** [comment]
Hello, do you still have docker to download?

**Comment by gg22mm** [comment]
Why can't anyone answer? Is this project abandoned？
